compiler/rustc_lint_defs/src/builtin.rs

```diff
@@ -3793,7 +3793,7 @@ declare_lint! {
     /// ```rust,compile_fail
     /// #![deny(text_direction_codepoint_in_comment)]
     /// fn main() {
-    ///     println!("{:?}"); // '‮');
+    #[doc = "    println!(\"{:?}\"); // '\u{202E}');"]
```

Comment:
why should we use `#[doc = "..."]` as the example?

Reply:
The example isn't changed here. The attribute is used so that the RLO can be escaped. This is to avoid having to set `#![allow(text_direction_codepoint_in_literal)]` on the whole crate because the allow attribute no longer works at the item level.